src/store/index.js

```diff
@@ -15,6 +16,7 @@ export const store = new Vuex.Store({
   modules: {
     root,
     user,
+    Upload,
```

Comment:
@MoranLi Can you please add this Upload module, into the hot-reloading section on line 27, 32, and 38 of this file. You can see how the slides module is imported on line 27, 32, and 38 of this file.

Reply:
really sorry forget add that, should be fine now, the sample upload page should be removed now